src/main/java/no/nav/familie/inntektsmelding/forespørsel/tjenester/ForespørselBehandlingTjeneste.java

```diff
@@ -123,11 +129,7 @@ public class ForespørselBehandlingTjeneste {
         // Forespørsler som skal opprettes
         var skalOpprettes = utledNyeForespørsler(forespørsler, eksisterendeForespørsler);
         for (OppdaterForespørselDto forespørselDto : skalOpprettes) {
-            var opprettForespørselTask = OpprettForespørselTask.lagTaskData(ytelsetype,
-                aktørId,
-                saksnummer,
-                forespørselDto.orgnr(),
-                forespørselDto.skjæringstidspunkt());
+            var opprettForespørselTask = lagOpprettForespørselTaskData(ytelsetype, aktørId, saksnummer, forespørselDto);
```

Comment:
Du kunne beholdt den statiske metoden på tasken og gjort det samme der. Når har du jo trukket inn ObjectMapper der og. 
Ikke at det nødvendigvis er det beste, men en fordel med det er at tasken selv tar ansvar for hvordan data den ønsker å få inn.